src/components/container/list/list.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import styles from './list.module.css';
 
+var key
+var size = localStorage.length
+
+function GetList(){
+    var listELements = []
```

Comment:
const listElements = [ ]; tu warto by wrzucić listę po której będziesz iterować czyli np. jeśli trzymasz w localStorage tablicę obiektów reprezentujących poszczególne elementy listy to zaciągnij z localStorage całą tablicę tutaj, np const listElements = [ ...localStorage.getItem('dataKey')];

najlepiej byłoby jednak wyciągnąc taką tablicę do góry i umieścić ją w stanie komponentu: 

```
const List = () => {
const [itemsList, setItemsList] = useState([]);

useEffect(() => {
  setItemsList(localStorage.getItem('dataKey'));
}, [])

return (
<ul>
{itemsList.map((el, idx) => (
<li key={idx}> <button ...>{ el } </button></li>
)}
</ul>
);
};
```

czyli pozbywamy się również funkcji getList która jest tutaj niepotrzebna